handsontable/src/plugins/collapsibleColumns/collapsibleColumns.js

```diff
@@ -69,6 +70,35 @@ const actionDictionary = new Map([
  *   ],
  * });
  * ```
+ * :::
+ *
+ * ::: only-for react
+ * ```jsx
+ * <HotTable
+ *   data={generateDataObj()}
+ *   colHeaders={true}
+ *   rowHeaders={true}
+ *   // @ts-ignore
```

Comment:
This doesn't look too good, why is it here?

Reply:
As I wrote in the PR description: `I added // @ts-ignore in the places where TS show errors`. I haven't extended the task scope. It seems that there is problem with type definition.

Reply:
I agree with @jansiegel it doesn't look good. If there are some issues with TS I would create a separate issue and here, I would remove all `// @ts-ignore` comments.

Reply:
@kirszenbaum WDYT about changing the [`NestedHeader` description](https://handsontable.com/docs/api/nested-headers/#options) to add information about `boolean` as an option? Could you help me with that?